webpack.config.js

```diff
@@ -1,11 +1,22 @@
 const path = require('path');
 
-module.exports = [
-    buildJsFromTo({from: 'plugins/plugin.archive_analytics.js', to: 'BookReader/plugins/plugin.archive_analytics.js'}),
-    buildJsFromTo({from: 'plugins/menu_toggle/plugin.menu_toggle.js', to: 'BookReader/plugins/plugin.menu_toggle.js'}),
-    buildJsFromTo({from: 'plugins/tts/plugin.tts.js', to: 'BookReader/plugins/plugin.tts.js'}),
+const listOfFiles = [
+    'plugins/plugin.archive_analytics.js',
+    'plugins/menu_toggle/plugin.menu_toggle.js',
+    'plugins/tts/plugin.tts.js',
 ];
```

Comment:
Let's keep the exports at the top here; otherwise it's a lot of jumping around to see what's happening or where the important stuff is happening.

Also, saving the list of files in an array doesn't really add anything in this case; it's useful if we were going to be re-using it, or if the variable name helped make the code clearer, but placing the function here makes this self-documenting, since the name of the function explains what the list is.

```suggestion
module.exports = buildJsFiles([
    'plugins/plugin.archive_analytics.js',
    'plugins/menu_toggle/plugin.menu_toggle.js',
    'plugins/tts/plugin.tts.js',
]);
```

Reply:
yes totally 👍  - I was using es6 arrows. will hoist ⬆️ to `function`

Reply:
Ahhh, gotcha; I forgot they don't hoist! I was wondering why that was :P

Reply:
done